vars/terraform.groovy

```diff
@@ -98,7 +98,17 @@ def call(userConfig = [:]) {
               ]) {
                 scmOutput = getInfraSharedTools(sharedToolsSubDir)
 
+                // Retrieve published reports for idempotency (otherwise Terraform's "local_file" will be marked as always re-created as ignored from the SCM)
+                // Note: always run after call to getInfraSharedTools() (or it will be overridden)
+                if(finalConfig.publishReports && finalConfig.publishReports.size > 0) {
+                  for (int i = 0; i < finalConfig.publishReports.size; i++) {
+                    final String fileRelPath = finalConfig.publishReports[i]
+                    file_text = new URL ("https://reports.jenkins.io/${fileRelPath}").getText()
+                    writeFile(file: fileRelPath, text: file_text)
```

Comment:
```suggestion
                    final String relativeFilePath = finalConfig.publishReports[i]
                    file_text = new URL ("https://reports.jenkins.io/${relativeFilePath}").getText()
                    writeFile(file: relativeFilePath, text: file_text)
```
Changed the order for a proper English sounding name, and not abbreviating it. (nit, can be abbreviated if preferred)

Reply:
the new variable name is better to me